x-pack/plugins/data_usage/public/app/components/data_usage_metrics.tsx

```diff
@@ -204,13 +215,14 @@ export const DataUsageMetrics = memo(
             data-test-subj={getTestId('filter')}
           />
         </FlexItemWithCss>
-
         <FlexItemWithCss>
-          {isFetched && data ? (
-            <Charts data={data} data-test-subj={dataTestSubj} />
+          {hasFetchedDataUsageMetricsData && usageMetricsData ? (
+            <Charts data={usageMetricsData} data-test-subj={dataTestSubj} />
           ) : isFetching ? (
-            <EuiLoadingElastic data-test-subj={getTestId('charts-loading')} />
-          ) : null}
+            <ChartsLoading data-test-subj={dataTestSubj} />
+          ) : (
+            <NoDataCallout data-test-subj={dataTestSubj} />
```

Comment:
shows no data callout instead of nothing. We did this as we now allow deselecting all data streams. We don't send a network request in this case though!